pod/perldelta.pod

```diff
@@ -411,6 +411,16 @@ This prevents integer overflows when appending to a large C<SV> for
 C<readpipe> aka C<qx//> and C<readline>.
 L<https://www.perlmonks.org/?node_id=11161665>
 
+=item *
```

Comment:
commit message: the commit hashes won't be valid once the PR is merged (we rebase and merge)